app/models/tag.rb

```diff
@@ -1,29 +1,19 @@
 class Tag < ActiveRecord::Base
   has_many :taggings
+  has_many :tagables, through: :taggings, source: :tagable, source_type: Tagable
+
+  # creates polymorphic joins for all tagable classes
+  Tagable.classes.each do |klass|
```

Comment:
meta programming!!!!

a comment here explaining that this gives tags:

tag#entities
tag#relationships
tag#lists 

could be helpful for future littlesis hackers...

Reply:
+1